recipes/sigmut/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "sigmut" %}
+{% set version = "1.0" %}
+{% set sha256 = "d0060805ffb819461ad5666be53f11879ba0ec37a79496bda6bb1af32f8c02b8" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/lgpdv/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+
+requirements:
+  host:
+    - python >=3.7
```

Comment:
Are all of these host requirements? Usually host requirements for noarch python builds are only python and pip.

Reply:
Yes they are. What is weird is that locally I can create the conda repository and upload without any problem to my channel (conda install -c lgpdv sigmut).

Reply:
Hello @CJ-Wright . Finally all tests have succeeded. I think the PR is ready to be merged now.